### ISSUE TITLE
provide an API interface

### ISSUE THREAD

**Issue by rich5art** [ThePacielloGroup/CCAe#151]
We would like to use Colour Contrast Analyzer in our automated tests.  Is there an interface that would allow a user to send Hex colour codes to this application and have it return pass/fail values?

Today we are doing this manually.  Leveraging this tool in automation would allow us to catch failures earlier in the process.  It would also allow us to do more thorough testing with a significant gain in efficiency.

**Comment by ferllings** [comment]
No there no such API.
But that's interesting.
How do you see that? how do you see it fit in your process? like a CLI version?

**Comment by patrickhlauke** [comment]
there are already various tools like https://www.npmjs.com/package/color-contrast-checker that can be integrated into CI chains ... it'd probably be easier just to use something like that
